packages/create-redwood-app/templates/ts/web/tsconfig.json

```diff
@@ -35,6 +35,7 @@
   "include": [
     "src",
     "config",
+    ".storybook/*",
```

Comment:
I said `./.storybook/**/*` in the upgrade guide. Is what you have here better?

Reply:
./.storybook/* is absolutely fine

Reply:
I agree with you both :smile: 

In practice, >99 % of people will only ever have two or three files directly in that folder … definitely works for me :sunglasses: . But then someone *will* come along and fan out their custom configuration into a sub-directory and we're back to square one again.

So i'm updating this PR to match the upgrade guides version.